tools/biopattern.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/python
+# @lint-avoid-python-3-compatibility-imports
+#
+# biopattern - Identify random/sequential disk access patterns.
+#              For Linux, uses BCC, eBPF.
+#
+# Copyright (c) 2022 rockyxing.
```

Comment:
Could you proper spell name here, "Feng Xing" or "Rocky Xing" or anything your prefer? You can check biolatency.py for an example.

Reply:
Done